tools/release-posts/templates/release.ejs

```diff
@@ -11,25 +11,13 @@
 <!-- /wp:paragraph -->
 
 <!-- wp:list -->
-<ul>
-  <% for ( const packageName in contributors ) { %>    
-    <% const {org, repo, contributors: packageContributors, totalCommits, baseRef, headRef} = contributors[packageName] %>
-    <% if ( packageContributors ) { %>
-      <li>
-        <a target="_blank" href="https://github.com/<%= org %>/<%= repo %>/compare/<%= baseRef %>...<%= headRef %>">
-          <%= totalCommits %> commits from <%= packageContributors.length %> contributors
-        </a> 
-        in <%= packageName %>.
-      </li>
-    <% } %>
-  <% } %>
-</ul>
+<ul><% for ( const packageName in contributors ) { %><% const {org, repo, contributors: packageContributors, totalCommits, baseRef, headRef} = contributors[packageName] %><% if ( packageContributors ) { %><li><a href="https://github.com/<%= org %>/<%= repo %>/compare/<%= baseRef %>...<%= headRef %>"><%= totalCommits %> commits from <%= packageContributors.length %> contributors</a> in <%= packageName %>.</li><% } %><% } %></ul>
```

Comment:
This formatting is not good, we need the ejs to be easy to edit and maintain and read, I use this extension for ejs support in VSCode https://marketplace.visualstudio.com/items?itemName=DigitalBrainstem.javascript-ejs-support

Reply:
@samueljseay I deliberately formatted things this way because with the old, more readable formatting, pasting into Gutenberg didn't work for me. I didn't like all those new lines in all the places. Maybe there's a different way to change the output here so that it works for copy and paste?

Reply:
@lsinger damn, thats frustrating :/ that seems like a sort of big issue, because ideally we would have nice readable/maintainable templates (that was why I moved away from the string concatenation mess we had in the past)

I think we have 2 things available in EJS we should try before making templates harder to read.

1. pass the option `rmWhitespace` to the render call (see `Options` [here](https://ejs.co/)) - you can pass it to `renderFile` [here](https://github.com/woocommerce/woocommerce/blob/5ba15f84cd39a36374d0fb0c669abd1a6d5cef41/tools/release-posts/lib/render-template.ts#L17).

2. There are also whitespace slurping tags: `<%_` and `_%>` but probably just doing rmWhitespace makes more sense.